### PR TITLE
Allow to create attachments

### PR DESCRIPTION
**Pull request by barmac**
Related to https://github.com/camunda/camunda-modeler/issues/1462

### CODE REVIEW

File: lib/features/rules/BpmnRules.js [hunk 1]
```diff
@@ -128,6 +128,10 @@ BpmnRules.prototype.init = function() {
         return canConnect(element.source, element.target, element);
       }
 
+      if (element.host) {
+        return !!canAttach(element, element.host, null, position);
```
Comment (nikku) [inline lib/features/rules/BpmnRules.js]: Can we drop the `!!` here?

File: test/spec/features/rules/BpmnRulesSpec.js [hunk 2]
```diff
@@ -81,6 +81,36 @@ describe('features/modeling/rules - BpmnRules', function() {
...
+      var task = elementFactory.createShape({ type: 'bpmn:Task' }),
+          boundaryEvent = elementFactory.createShape({
+            type: 'bpmn:BoundaryEvent',
+            eventDefinitionType: 'bpmn:EscalationEventDefinition',
+            host: task
+          });
+
+      // then
+      expectCanCreate([ task, boundaryEvent ], 'Process', true);
+    }));
```
Comment (nikku) [inline test/spec/features/rules/BpmnRulesSpec.js]: Add space.

File: lib/features/rules/BpmnRules.js (hunk 1 above)
Comment (barmac) [reply lib/features/rules/BpmnRules.js]: Yes, as soon as we merge https://github.com/bpmn-io/min-dash/pull/14 and update `min-dash`. Otherwise, we need to somehow handle the case that the rule might return `"attach"` or `true` based only on the selection order.

File: test/spec/features/rules/BpmnRulesSpec.js (hunk 2 above)
Comment (barmac) [reply test/spec/features/rules/BpmnRulesSpec.js]: Not sure where.

**Comment by barmac** [comment]
> Can you update the message with a `rules` target:
> 
> ```
> fix(rules): allow ...
> ```

Done.

File: package.json [hunk 3]
```diff
@@ -45,7 +45,7 @@
     "cpx": "^1.5.0",
     "del": "^3.0.0",
     "eslint": "^5.16.0",
-    "eslint-plugin-bpmn-io": "^0.8.2",
+    "eslint-plugin-bpmn-io": "github:bpmn-io/eslint-plugin-bpmn-io#5-require-empty-lines-before-comments",
```
Comment (nikku) [inline package.json]: Can we revert this change?
